### ISSUE TITLE
Confusing upgrade instructions about avoiding public/packs

### ISSUE THREAD

**Issue by januszm** [shakacode/shakapacker#159]
The instructions on this page 
https://github.com/shakacode/shakapacker/blob/master/docs/v6_upgrade.md#how-to-upgrade-to-webpacker-v600rc6-from-v5

suggest moving js files from packs to `app/javascript` and thus not using packs. This causes js and css files to be precompiled in `public/assets` but at the same time, if I'm not mistaken, it removes the `public/assets/packs` directory (it's not created). The manifest.json file is missing, there's a manifes.js file, which is empty and is in `public/assets`, while somewhere in the code `manifest.json` is still expected to be generated in `public/packs/` . This causes the following error:

```
ActionView::Template::Error
  (Shakapacker can't find application.js in /var/app/current/public/packs/manifest.json. Possible causes: ...
```

To sum up, with my configuration, after upgrading from webpacker with Rails 6 to shakapacker with Rails 7, the manifest.json file is missing. There's a manifest.js file which is empty and the app won't work in production mode. precompiled files are in public/assets, packs is missing.

```yaml
# webpacker.yml
default: &default
  source_path: app/javascript
  source_entry_path: /
  nested_entries: false

  public_root_path: public
  public_output_path: packs
  cache_path: tmp/webpacker
  webpack_compile_output: true
  webpacker_precompile: true
# ...
production:
  <<: *default
  compile: false
  cache_manifest: true
```

**Comment by januszm** [comment]
UPDATE: apparently the error itself was caused by broken package.json settings, I had a package in devDepencencies that was actually present somewhere in production code. It was quite hard to debug though.

**Comment by justin808** [comment]
@januszm does this help: https://github.com/shakacode/shakapacker/pull/160

**Comment by januszm** [comment]
@justin808  It looks good, there's less emphasis on moving files out of packs while  still enough info about not nesting. 👍
